requirements.txt

```diff
@@ -22,5 +22,5 @@ PyYAML==6.0.1
 securetar==2023.3.0
 sentry-sdk==1.34.0
 voluptuous==0.13.1
-dbus-fast==2.12.0
```

Comment:
TODO: revert this if CI passes as it should be done via https://github.com/home-assistant/supervisor/pull/4681